packages/usa-card/src/test/test-patterns/test-usa-card.json

```diff
@@ -0,0 +1,98 @@
+      "cta_text": "Express gentleness"
+    },
+    {
+      "title": "He never smiled, he never frowned",
+      "classes_override": "tablet:grid-col-4",
+      "header_first": true,
+      "media": {
+        "media_classes": "usa-card__media--exdent usa-card__media--fix-aspect add-aspect-1x1"
+      },
+      "content": "Simon Wheeler backed me into a corner and blockaded me there with his chair, and then sat me down and reeled off the monotonous narrative which follows this paragraph. He never smiled, he never frowned, he never changed his voice from the gentle-flowing key to which he tuned the initial sentence, he never betrayed the slightest suspicion of enthusiasm; but all through the interminable narrative there ran a vein of impressive earnestness and sincerity, which showed me plainly that, so far from his imagining that there was any thing ridiculous or funny about his story, he regarded it as a really important matter, and admired its two heroes as men of transcendent genius in finesse.",
+      "cta_text": "Rouse Simon"
+    },
+    {
+      "title": "Garrulous old Simon Wheeler",
+      "classes_override": "tablet:grid-col-4",
+      "header_first": false,
+      "media": {
+        "media_classes": "usa-card__media--inset"
+      },
+      "content": "Simon Wheeler backed me into a corner and blockaded me there with his chair, and then sat me down and reeled off the monotonous narrative which follows this paragraph. He never smiled, he never frowned, he never changed his voice from the gentle-flowing key to which he tuned the initial sentence, he never betrayed the slightest suspicion of enthusiasm; but all through the interminable narrative there ran a vein of impressive earnestness and sincerity, which showed me plainly that, so far from his imagining that there was any thing ridiculous or funny about his story, he regarded it as a really important matter, and admired its two heroes as men of transcendent genius in finesse.",
+      "cta_text": "Read More"
+    },
+    {
+      "title": "He never smiled, he never frowned",
+      "classes_override": "tablet:grid-col-4",
+      "header_first": true,
+      "media": {
+        "media_classes": "usa-card__media--inset"
+      },
+      "content": "Simon Wheeler backed me into a corner and blockaded me there with his chair, and then sat me down and reeled off the monotonous narrative which follows this paragraph. He never smiled, he never frowned, he never changed his voice from the gentle-flowing key to which he tuned the initial sentence, he never betrayed the slightest suspicion of enthusiasm; but all through the interminable narrative there ran a vein of impressive earnestness and sincerity, which showed me plainly that, so far from his imagining that there was any thing ridiculous or funny about his story, he regarded it as a really important matter, and admired its two heroes as men of transcendent genius in finesse.",
+      "cta_text": "Read More"
+    }
+  ]
+}
```

Comment:
Discovered some presentation issues with the card test:
![image](https://user-images.githubusercontent.com/93996430/162757457-b1f6c359-0096-4064-9afc-0dffee488230.png)
Will investigate what is happening and if it is a data or styles issue.